api/helper.py

```diff
@@ -47,7 +47,7 @@ def user_register_create_team_and_players(user, team_name, team_country):
 """
 
 
-def transfer_list_name_instead_of_id(instance):
+def show_transfer_list(instance):
```

Comment:
function name is already changed in PR https://github.com/i-huzaifa-arshad/Soccer-Game-API/pull/8/files . It will create conflicts later. remove changes from this PR